girder/models/setting.py

```diff
@@ -279,6 +279,12 @@ def validateCoreEmailFromAddress(doc):
             raise ValidationException('Email from address must not be blank.', 'value')
 
     @staticmethod
+    @setting_utilities.validator(SettingKey.CONTACT_EMAIL_ADDRESS)
+    def validateCoreContactEmailAddress(doc):
+        if not doc['value']:
+            raise ValidationException('Email from address must not be blank.', 'value')
```

Comment:
There is a configuration option for specifying a regex email addresses must conform to, that can probably be used here as well.